jobs/openshift/cluster/brew/Jenkinsfile

```diff
@@ -0,0 +1,143 @@
+            string(defaultValue: (options.installationGitUrl ?: 'https://github.com/integr8ly/installation.git'), description: 'Integreatly installer Git URL', name: 'installationGitUrl'),
+            string(defaultValue: (options.installationGitBranch ?: 'master'), description: 'Integreatly installer Git branch', name: 'installationGitBranch'),
+        ]
+        options.openshiftMasterUrl = userInput.openshiftMasterUrl
+        options.clusterAdminUsername = userInput.clusterAdminUsername
+        options.clusterAdminPassword = userInput.clusterAdminPassword
+        options.installationGitUrl = userInput.installationGitUrl
+        options.installationGitBranch = userInput.installationGitBranch
+        verifyImageSyncOptions(options)
+    }
+}
+
+def setClusterAdminCredentials() {
+    def integreatlyCredentialsID = 'tower-openshift-cluster-credentials'
+    def clusterAdminCredentials = [:]
+    withCredentials([usernamePassword(credentialsId: integreatlyCredentialsID, usernameVariable: 'CLUSTER_ADMIN_USERNAME', passwordVariable: 'CLUSTER_ADMIN_PASSWORD')]) {
+        clusterAdminCredentials.clusterAdminUsername = "${CLUSTER_ADMIN_USERNAME}"
+        clusterAdminCredentials.clusterAdminPassword = "${CLUSTER_ADMIN_PASSWORD}"
+    }
+
+    return clusterAdminCredentials
+}
+
+String getHost(String url) {
+    URI uri = new URI(url);
+    String[] urlParts = uri.getHost().split(/\./).takeRight(3)
+    return urlParts.join('.')
+}
+
+stage('Image Sync Options') {
+    def clusterAdminCredentials = setClusterAdminCredentials()
+    imageSyncOptions.openshiftMasterUrl = params.openshiftMasterUrl
+    imageSyncOptions.clusterAdminUsername = params.clusterAdminUsername ?: clusterAdminCredentials.clusterAdminUsername // Defaults to the username stored in tower-openshift-cluster-credentials
+    imageSyncOptions.clusterAdminPassword = params.clusterAdminPassword ?: clusterAdminCredentials.clusterAdminPassword // Defaults to the password stored in tower-openshift-cluster-credentials
+    imageSyncOptions.installationGitUrl = params.installationGitUrl
+    imageSyncOptions.installationGitBranch = params.installationGitBranch
+    verifyImageSyncOptions(imageSyncOptions)
+    
+    println "Image Sync Options: ${imageSyncOptions}"
+    currentBuild.displayName = "${currentBuild.displayName} ${imageSyncOptions.installationGitBranch}"
+    currentBuild.description = imageSyncOptions.openshiftMasterUrl
+}
+
+def githubSSHCredentialsID = 'jenkinsgithub'
+def imageSyncConfigFile = './integreatly.yml'
+String brewContainerRegistry
+def projectsToSync
+def syncImages = false
+
+node('cirhos_rhel7') {
+    cleanWs()
+    stage('Ensure Sync File Exists') {
+        dir('installation') {
+            gitCheckoutRepo(imageSyncOptions.installationGitUrl, imageSyncOptions.installationGitBranch, githubSSHCredentialsID, '.')
+            syncImages = fileExists imageSyncConfigFile
```

Comment:
This is setting syncImages to true if the file "integreatly.yml" exists, but should it not also check to see if there is a "syncImages" section in the file?